addons/web/static/tests/search/search_bar.test.js

```diff
@@ -845,6 +845,7 @@ test("should wait label promises for one2many search defaults", async () => {
 
     def.resolve();
     await animationFrame();
+    await animationFrame();
```

Comment:
ok, I don't understand anymore

Reply:
but the additional `await animationFrame();` should be done in between `mountWithSearch` and the ` expect().toHaveCount(0)`;

Reply:
The deferred blocks the first rendering, so I think it would not be useful.